go-controller/pkg/types/const.go

```diff
@@ -60,6 +60,7 @@ const (
 	DefaultDenyPriority = 1000
 
 	// priority of logical router policies on the OVNClusterRouter
+	DefaultEgressFirewallPriority         = "10001"
```

Comment:
I'm not sure this name is suitable. Shouldn't it be like MaxEgressFirewallPriority? Also the EgressFirewallStartPriority should have a comment above it maybe describing that egress firewall priority's are decremented (I forget exactly how this works).

Reply:
OK, I will add some comments.